### PR TITLE
MCD migration - Add details to Exchange Online pre-work

### PR DESCRIPTION
**Pull request by heikobischoff**

### CODE REVIEW

**Comment by JoeDavies-MSFT** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md [hunk 1]
```diff
@@ -115,16 +115,28 @@ Read and apply the [ADFS Migration steps](ms-cloud-germany-transition-add-adfs.m
...
-||||
+Availability configuration needs to transition to use the [Office 365 worldwide endpoints](/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide) when Exchange Online migration is completed.
+Failure to do so may result in service or client failure at a later phase of customer migration.
+
+### Exchange Online customers using POP3/IMAP4
+**Applies to**: Exchange Online customers using POP3/IMAP4 protocols to connect their client devices to Exchange Online<br>
+**When applied**: Any time before phase 5 starts
+
+Users using device connections to Microsoft Cloud Deutschland endpoints for client protocols IMAP4, POP3, SMTP are required to manually update their client devices to switch to the [Exchange Online server names](/exchange/clients-and-mobile-in-exchange-online/pop3-and-imap4/pop3-and-imap4#settings-users-use-to-set-up-pop3-or-imap4-access-to-their-exchange-online-mailboxes) if they want to pursue with these protocols.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md]:
```suggestion
Users using device connections to Microsoft Cloud Deutschland endpoints for client protocols IMAP4, POP3, and SMTP are required to manually update their client devices to switch to the [Exchange Online server names](/exchange/clients-and-mobile-in-exchange-online/pop3-and-imap4/pop3-and-imap4#settings-users-use-to-set-up-pop3-or-imap4-access-to-their-exchange-online-mailboxes) if they want to pursue these protocols.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md [hunk 2]
```diff
@@ -115,16 +115,28 @@ Read and apply the [ADFS Migration steps](ms-cloud-germany-transition-add-adfs.m
...
+
+Pre-communicate this dependency to users of these protocols and ensure they either switch to use Outlook mobile or Outlook on the web during this migration. Failure to update the client endpoints will result in client connection failures with Microsoft Cloud Deutschland when mailboxes are migrated.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md]:
```suggestion
Pre-communicate this dependency to users of these protocols and ensure they either switch to use Outlook for mobile or Outlook on the web during this migration. Failure to update the client endpoints will result in client connection failures with Microsoft Cloud Deutschland when mailboxes are migrated.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md [hunk 3]
```diff
@@ -115,16 +115,28 @@ Read and apply the [ADFS Migration steps](ms-cloud-germany-transition-add-adfs.m
...
+
+The exact point of time to change the endpoint configuration cannot be predicted. During phase 5, user mailboxes will be successively migrated to the new Office 365 German region. The start of phase 5 will be announced in the Office 365 Admin Center. After each individual mailbox migration, the old endpoints become unavailable immediately, the client device will be disconnected from Exchange Online and the user must reconfigure all connected devices with the new [Exchange Online server endpoint](/exchange/clients-and-mobile-in-exchange-online/pop3-and-imap4/pop3-and-imap4#settings-users-use-to-set-up-pop3-or-imap4-access-to-their-exchange-online-mailboxes). 
+
+The new endpoints cannot be configured in advance, they  become available **only** when the mailbox is migrated.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md]:
```suggestion
The new endpoints cannot be configured in advance; they become available **only** when the mailbox is migrated.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md [hunk 4]
```diff
@@ -115,16 +115,28 @@ Read and apply the [ADFS Migration steps](ms-cloud-germany-transition-add-adfs.m
...
+
+Pre-communicate this dependency to users of these protocols and ensure they either switch to use Outlook mobile or Outlook on the web during this migration. Failure to update the client endpoints will result in client connection failures with Microsoft Cloud Deutschland when mailboxes are migrated.
+
+The exact point of time to change the endpoint configuration cannot be predicted. During phase 5, user mailboxes will be successively migrated to the new Office 365 German region. The start of phase 5 will be announced in the Office 365 Admin Center. After each individual mailbox migration, the old endpoints become unavailable immediately, the client device will be disconnected from Exchange Online and the user must reconfigure all connected devices with the new [Exchange Online server endpoint](/exchange/clients-and-mobile-in-exchange-online/pop3-and-imap4/pop3-and-imap4#settings-users-use-to-set-up-pop3-or-imap4-access-to-their-exchange-online-mailboxes).
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md]:
```suggestion
The exact point of time to change the endpoint configuration cannot be predicted. During phase 5, user mailboxes will be successively migrated to the new Office 365 German region. The start of phase 5 will be announced in the Office 365 Admin Center. After each individual mailbox migration, the old endpoints become unavailable immediately, the client device will be disconnected from Exchange Online, and the user must reconfigure all connected devices with the new [Exchange Online server endpoint](/exchange/clients-and-mobile-in-exchange-online/pop3-and-imap4/pop3-and-imap4#settings-users-use-to-set-up-pop3-or-imap4-access-to-their-exchange-online-mailboxes). 
```

File: microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md [hunk 5]
```diff
@@ -115,16 +115,28 @@ Read and apply the [ADFS Migration steps](ms-cloud-germany-transition-add-adfs.m
...
+
+The exact point of time to change the endpoint configuration cannot be predicted. During phase 5, user mailboxes will be successively migrated to the new Office 365 German region. The start of phase 5 will be announced in the Office 365 Admin Center. After each individual mailbox migration, the old endpoints become unavailable immediately, the client device will be disconnected from Exchange Online and the user must reconfigure all connected devices with the new [Exchange Online server endpoint](/exchange/clients-and-mobile-in-exchange-online/pop3-and-imap4/pop3-and-imap4#settings-users-use-to-set-up-pop3-or-imap4-access-to-their-exchange-online-mailboxes). 
+
+The new endpoints cannot be configured in advance, they  become available **only** when the mailbox is migrated.
+
+This limitation is unique to POP3/IMAP4 connections and does not apply to MAPI connections. Consider switching to Outlook apps or other modern protocols on all connected devices before phase 5 starts to improve the user experience.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md:139]:
```suggestion
This limitation is unique to POP3/IMAP4 connections and does not apply to MAPI connections. Consider switching to Outlook apps or other modern protocols on all connected devices, before phase 5 starts, to improve the user experience.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor suggestions.

**Comment by heikobischoff** [comment]
Commited, thanks @JohanFreelancer9!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JoeDavies-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@JoeDavies-MSFT Please merge this PR. Thanks.

**Pull request merged**